src/classes/TinySocket.ts

```diff
@@ -66,32 +66,40 @@ export class TinySocket {
   }
 
   wrap<T>(
-    method: (callback: (error?: Error, value?: T) => void) => void,
+    method: (
+      resolve: (value: T) => void,
+      reject: (error: Error) => void,
+    ) => void,
   ): Promise<T> {
     return new Promise((resolve, reject) => {
-      const callback = (error?: Error, value?: T) => {
+      const handle_timeout = () => {
```

Comment:
Tiny nit:

```suggestion
      const handleTimeout = () => {
```

Reply:
I fixed the style and also moved the logic of the unused `handleTimeout()` method into the wrapper. PTAL.